README.md

```diff
@@ -1,34 +1,16 @@
 # Descriptive-Data-Uploading-App
 
-## Set up the application
+Whenever it exports the zip folder from your source system, have your custom export app automatically run the DescriptiveDataUploadApp. Clone the DescriptiveDataUploadApp to your machine by running the following command: `git clone https://github.com/microsoft/vivainsights_ingressupload.git`.
```

Comment:
I still feel like it is up to them - whether they want to automate or not. We just provide the app that can upload a zipped file and return results. It is up to them whether they want an app they build that calls this or they have someone manually running this app every time their zipped file is updated. Thoughts?